package.json

```diff
@@ -51,10 +51,11 @@
     "gulp": "^4.0.2",
     "gulp-run": "^1.7.1",
     "gulp-string-replace": "^1.1.2",
+    "husky": "^4.0.10",
```

Comment:
It looks like the Composer `cghooks` package isn't compatible with PHP 5.6. That was used to install the pre-commit hook.

So this an alternative to that.